drivers/backup/backup.go

```diff
@@ -355,6 +355,9 @@ type Rule interface {
 	// CreateRuleForBackup creates backup rule
 	CreateRuleForBackup(appName string, orgID string, prePostFlag string) (bool, string, error)
 
+	// CreateRuleForBackupWithMultipleApplications creates backup rule for multiple application
+	CreateRuleForBackupWithMultipleApplications(ruleName string, orgID string, appList []string, prePostFlag string, ctx context.Context) (bool, string, error)
```

Comment:
How about we pass `AppParameters` instead of `appList`? Then, we would not have to define `AppParametersNew`.